lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -1722,7 +1728,7 @@ def test_invalidate_sale_record(self):
         for i in range(2):
             course_registration_code = CourseRegistrationCode(
                 code='sale_invoice{}'.format(i), course_id=self.course.id.to_deprecated_string(),
-                created_by=self.instructor, invoice=self.sale_invoice_1, mode_slug='honor'
+                created_by=self.instructor, invoice_item=self.invoice_item, mode_slug='honor'
```

Comment:
hrm. This is what I'm a bit afraid in having both an invoice and invoice_item FK's (see below). @stephensanchez @wedaly can we discuss? My gut feeling is that we need to set both as I'm not sure what the downstream effects might be with regards to other areas of the code (particularly reporting) if invoice is not always consistently set...